### PR TITLE
Closes #1808: Add documentation for ror

### LINKED ISSUE #1808: Make documentation for API endpoint for ROR upload

### PR DESCRIPTION
**Pull request by madryk**
fix ror schema so its compatible with solr 7.3.1

### CODE REVIEW

File: doc/sphinx-guides/source/admin/ror.rst [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+===
+
+.. contents:: Contents:
+    :local:
+
+About ROR
+---------
+
+ROR (Research Organization Registry) is a registry of identifiers for research
+organizations. ROR identifiers helps with matching affiliation data with institutions.
```
Comment (dbojanek) [inline doc/sphinx-guides/source/admin/ror.rst]:
```suggestion
organizations. ROR identifiers help with matching affiliation data with institutions.
```

File: doc/sphinx-guides/source/admin/ror.rst [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+For more informations visit `official ROR homepage <https://ror.org/>`_
+
+
+ROR in Dataverse
+----------------
+
+Dataverse uses ROR identifiers to enrich DataCite metadata format. In default dataset metadata fields
+configuration there is also turned on suggesting of ROR identifiers based on name of the organization.
```
Comment (dbojanek) [inline doc/sphinx-guides/source/admin/ror.rst]: I'd rephrase the second sentence a bit. Something like: `In dataset metadata fields configuration there is an option for suggesting ROR identifiers based on organization name (turned on by default).`

File: doc/sphinx-guides/source/admin/ror.rst [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+~~~~~~~~~~~~
+
+It is possible to use the same REST api enpoint as in full dump case to upload only selected ROR identifiers.
+In such cases you will need to prepare the dump data yourself. We will show an example how it can be achieved.
+
+Dump can be prepared using ROR api:
+
+.. literalinclude:: ../_static/util/make_ror_dump.sh
+
+In example above we include ROR identifiers for institutions from `Great Britain` with type `Education`.
```
Comment (dbojanek) [inline doc/sphinx-guides/source/admin/ror.rst:51]: Maybe add a sentence like: `Modify make_ror_dump.sh to obtain the data you want`.

Comment (madryk) [reply doc/sphinx-guides/source/admin/ror.rst:51]: I will change it to: `You can modify ``ROR_FILTER`` variable in the script to obtain the data you want.` (I don't include filename because the script is embedeed on the page - user don't have a file it can download; he must do a copy+paste to use it)

**Review by dbojanek — APPROVED** [review]
